Pipeline changes to check if code is formatted using prettier (#9362)
microsoft/vscode-python: CODING_STANDARDS.md

### Style

> * `prettier` formatter must be used to format `typescript,javascript` code.

Reviewer: ```suggestion
* Use `prettier` to format `TypeScript` and `JavaScript` code.
```

> * `prettier` formatter must be used to format `typescript,javascript` code.

Reviewer 2: ```suggestion
* `prettier` formatter must be used to format `typescript, javascript` code.
```

Reviewer 2: Use @kimadeline 's edit.
